modes/patch-mode/patch-mode.lisp

```diff
@@ -10,10 +10,15 @@
 (define-attribute delete-line-attribute
   (t :foreground :base08))
 
+(define-attribute hunk-line-attribute
+  (:light :foreground "black" :background "lightgrey")
+  (:dark :foreground "grey" :background "black"))
```

Comment:
If you specify by base value, isn't it likely that the color will be appropriate for any theme?

How about `(t :foreground :base0C :bold t)`, for example?